skimage/transform/tests/test_warps.py

```diff
@@ -117,6 +117,32 @@ def test_homography():
     assert_almost_equal(x90, np.rot90(x))
 
 
+def test_warp_fit_output():
+    x = np.zeros((5, 5), dtype=np.double)
+    x[1, 1] = 1
+    theta = -np.pi / 4
+    M = np.array([[np.cos(theta), - np.sin(theta), 0],
```

Comment:
For sake of uniformity and clarity, please change this to
```py
[[np.cos(theta), - np.sin(theta), 0],
 [np.sin(theta),   np.cos(theta), 4],
 [0,               0,             1]]
```
as it is in your second test case.